src/edu/najah/cap/data/data_exporting/export/convert_data/creators/ZipCreator.java

```diff
@@ -0,0 +1,52 @@
+package edu.najah.cap.data.data_exporting.export.convert_data.creators;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class ZipCreator {
+    private static final Logger ZipCreatorLogger = Logger.getLogger(ZipCreator.class.getName());
+
+    public static void createZipFolder(String folderName, List<String> fileNames) {
```

Comment:
this function looks complex because of the nested try and for loop, it should be extracted